frank/fit.py

```diff
@@ -152,6 +152,11 @@ def parse_parameters(*args):
         err = ValueError("method should be 'Normal' or 'LogNormal'")
         raise err
 
+    if model['hyperparameters']['nonnegative'] and model['hyperparameters']['method'] != 'Normal':
+        err = ValueError("nonnegative should only be 'true' if method is 'Normal'")
+        raise err
+
+
```

Comment:
I just noticed this - is non-negative = True really an error for the log normal model? Conceptually it's not an error

Reply:
Sorry, I've just come back to this branch and seen these comments. You're right, I've updated in 2d610e3